lib/ReadsUtils/ReadsUtilsImpl.py

```diff
@@ -435,7 +493,18 @@ def interleave(self, source_obj_ref, source_obj_name, fwd_shock_filename,
                                                            rev_shock_node, rev_shock_filename])
                         error_message += 'Forward Path {}, Reverse Path {}.'
                         error_message_bindings.extend([fwdpath, revpath])
-                        raise ValueError(error_message.format(*error_message_bindings))
+                        
+                        if reads_source == 'web':
+                            error_message += 'Forward File URL {}, Reverse File URL {}.'
+                            error_message_bindings.extend([fwdsource, revsource])
+
+                        if reads_source == 'staging':
```

Comment:
Could you just set STAGING_FILE_PREFIX to something else prior to the test, or mock a fn that just returns the value of STAGING_FILE_PREFIX which is then used in _get_s_f?